stetho/src/main/java/com/facebook/stetho/inspector/elements/DOMProvider.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.elements;
+
+import javax.annotation.Nullable;
+
+// TODO: javadocs
```

Comment:
Prefer to remove this comment and leave a note for yourself out of band to follow-up.  It's unprofessional IMO to accidentally leave in a TODO like this.